### DISCUSSION TITLE
Proper flow of transactions

### DISCUSSION THREAD

**Discussion by thehijacker** [firefly-iii/firefly-iii#8170] [Firefly III]
It has been a good week since my Firefly III instance is up and starting fresh without imporing old transactions. I figured out most of the features and am also using them. So I have:

- Asset account: Main bank account, Revolut account, savings account, cash only account, ... So basicaly everything that holds money on.
- Expense accounts: To track expenses like per categories? Not quite sure I use this right. I am puttings all transactions destination accounts to expense accounts from asset account. Is this proper way to fill them?
- Revenue accounts: Now this is puzzling. When I get payed from my company I do a Revenue / income transaction directly to my main asset bank account.  Source is "cash". But here if I put revenue account as source then I get negative amount under those accounts. How exactly should this work then? I should put my paycheck money first to revenue account and then move this to my asset account? Not sure why and how to use this properly.
- Liabilities: Here is my mortage account that I am paying monthly. Value is how much I still own to the bank. But when I pay monthly installment of money, I need to transfer money to this account so the value of mortage liabitliy gets reduced?

Everything else is more or less clear. I am now using budgets to track how much I spent. Never done that before and it will be interesting to see how often I get over my set budget values.

I also have a piggy bank that is linked to my cash savings. Not yet sure for what I am saving for. Wife knows this the best :).

Finaly my finances will get all in one place with overview of them. Just need to convince my wife also to use it.

Thank you for all the tips and for such a gret and free tool to help us with our personal finances. I am buying you a beer :).

**Comment by JC5** [comment]
>  I am puttings all transactions destination accounts to expense accounts from asset account. 

Yes, it's wat I do.

> Revenue accounts: Now this is puzzling. When I get payed from my company I do a Revenue / income transaction directly to my main asset bank account. Source is "cash". But here if I put revenue account as source then I get negative amount under those accounts. How exactly should this work then? 

Just put the revenue account like the name of your company. It is correct that they are negative.

> I need to transfer money to this account so the value of mortage liabitliy gets reduced?

Yes.

**Comment by JC5** [comment]
And I'm happy to read that you like the app, thanks for your donation!
